src/page-response-impact/ResponseImpactDashboard.tsx

```diff
@@ -135,37 +213,103 @@ const ResponseImpactDashboard: React.FC = () => {
         <ValidDataWrapper facilities={facilities.data}>
           <PageContainer>
             <Column>
-              <ScenarioName>{scenario?.name}</ScenarioName>
-              <PageHeader>COVID-19 Response Impact as of [DATE]</PageHeader>
-              <SectionHeader>Safety of Overall Population</SectionHeader>
+              <Link to="/">
+                <BackDiv>
+                  <IconBack alt="back" src={iconBackSrc} />
+                  Back to model
+                </BackDiv>
+              </Link>
+              <PageHeader>
+                [State or County] COVID-19 Response Impact
+              </PageHeader>
+              <ReportDateDiv>Report generated on [DATE]</ReportDateDiv>
```

Comment:
[DATE] here is current date, FYI. You can just use this `{format(new Date(), "MMM dd, yyyy")}`

Reply:
@sokunthel thank you! made that change: ![image](https://user-images.githubusercontent.com/1372946/81334683-41e26080-905b-11ea-9c42-b56766d5d462.png)